docs/api/rebac/groups.mdx

```diff
@@ -6,60 +6,93 @@ title: Groups API
+ensuring consistent and manageable access control. See the [Groups API Redoc](https://api.permit.io/v2/redoc#tag/Groups-(Upcoming-Feature)) for more information.
 
-Let's say we want to represent a company and its employees as a group, and then assign all the employees access to a shared marketing resource (social-media) via the group.  We'll have a group named 'company' (Which is represented as a resource named 'company'), and an instance of a specific company 'Acme'.  Users are assigned to `Acme` via a role called `member`
-Additionally, we will have a resource-instance of type 'marketing' - 'social_media' which supports the role 'editor'.
-Once we assign the role 'editor' to  the `company` group, the following will by automated by API:
-- create role derivation from 'company#member' to 'marketing#editor'
-- create a relation between 'company' and 'marketing' with a relation between 'company_group'
-- create relationship between 'company#Acme' and 'marketing#social_media'.
 
-![](groups-diagram.png)
+- **Group Resource Instances, and Users**: Organize your resource instances and users into groups, allowing for simplified ReBAC permissions for:
 
-Then we will have group 'company' and every user who is a 'member' in this group will have the role 'editor' in the resource 'marketing#social_media'.
+    - **Users:** Users: Including users in a group allows you to assign an entire cluster of users with a single role (Instead of assigning roles to each user individually). 
+    You can set a role assigned to the group to be automatically assigned to all its members and revoked once their membership is removed.
+    
+    :::info Example:
+    If we create a marketing group and assign a `member` role to it, we then assign users to the Group via the `member` role. We can then derive other roles based on this assignment (See example below)
+    :::
+    
+    - **Resource Instances**: Grouping resources enables you to manage access control for multiple resources simultaneously, ensuring all included resources share the same permission configuration.
 
-### Assign users to a group:
+    :::info Example:
+    In our marketing group, we can create another group titled `social_media`. This group will contain resource instances of marketing materials. We can assign this newly created group the role of editor. 
+    The Groups API allows you to create a derivation between the `social_media#editor` role and the `marketing#member` role - meaning all users with the `marketing#member` role will automatically be assigned 
+    with the `social_media#editor` role.
+    :::
+    :::info Example:
+    New resources added to `social_media` will be accessible to edit by members of the marketing group, and new users added as members to the marketing group will automatically 
+    have editor access to social media assets.
+    :::
```

Comment:
This is not a new example - it's part of the previous one.
Not sure this needs to be an admonition at all